### ISSUE TITLE
Consider HiGHS for CI

### ISSUE THREAD

**Issue by freemin7** [matbesancon/NORBiP.jl#1]
HiGHS is a CI runnable solver

**Comment by matbesancon** [comment]
true, HiGHS could be considered, did you get a blocker with the default solvers used? SCIP and Cbc are both CI-compatible now

**Comment by freemin7** [comment]
I just read the ReadMe. I didn't run the code.
